lib/shoryuken/polling/weighted_round_robin.rb

```diff
@@ -36,12 +36,10 @@ def active_queues
       end
 
       def message_processed(queue)
-        return if @paused_queues.empty?
+        paused_queue = @paused_queues.find { |_time, name| name == queue }
+        return unless paused_queue
 
-        logger.debug "Unpausing #{queue}"
-        @paused_queues.reject! { |_time, name| name == queue }
-        @queues << queue
```

Comment:
When a queue gets [paused](https://github.com/ruby-shoryuken/shoryuken/pull/687/files#diff-4ecb88f1d35234e6a67d51dce18553f9915646a21cd18f43ecdaa01e4102948bR48) the queue is remove from `@queues`. This seems that it would break the activation of paused queues on `#message_processed`

Reply:
It's not clear from this diff alone, but I think `@queues` is now correctly updated in `unpause_queues`